app/assets/stylesheets/custom.css.scss

```diff
@@ -401,26 +433,39 @@ h4.state-of-equity {
   }
 }
 
-button {font-weight: 200;
+button {
+  font-weight: 500;
   text-transform: uppercase;
+  font-family: 'Oswald',sans-serif;
+  letter-spacing: 1px;
+  font-size: 1.0em;
 }
 
 .scroll_title {
-	font-family: 'Rokkitt', serif;
-	//font-weight: 700;
-	letter-spacing: 1px;
-	font-size: 18px;
-	line-height: 20px;
+  font-family: 'Rokkitt', serif;
+  //font-weight: 700;
+  letter-spacing: 1px;
+  font-size: 18px;
+  line-height: 20px;
 }
 
 li.group {
-	font-size: 14px;
+  font-size: 14px;
 }
 
 li.group a {
-	color: #656565;
+  color: #656565;

```

Comment:
Color literals like `#656565` should only be used in variable declarations; they should be referred to via variable everywhere else.